src/Order.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gett\MyparcelBE;
+
+use Gett\MyparcelBE\DeliveryOptions\DeliveryOptions;
+use MyParcelNL\Sdk\src\Adapter\DeliveryOptions\AbstractDeliveryOptionsAdapter;
```

Comment:
Santiy check: Aangezien dit uit een andere package komt. Dit gaat goed?

Reply:
Yes 😄 dat is onze beroemde php sdk